sdl_viewer/src/camera.rs

```diff
@@ -145,15 +185,13 @@ impl Camera {
     pub fn state(&self) -> State {
         State {
             transform: self.transform,
```

Comment:
Reviewers: should I also clean the angles from the struct State? this means that when someone has an old demo with saved positions, this will be completely uncompatible - if I leave like this, they could have some slight angle changes.